src/Microsoft.Owin.Security.Interop/AspNetTicketSerializer.cs

```diff
@@ -15,7 +15,7 @@ public class AspNetTicketSerializer : IDataSerializer<AuthenticationTicket>
         private const string DefaultStringPlaceholder = "\0";
         private const int FormatVersion = 5;
 
-        public static TicketSerializer Default { get; } = new TicketSerializer();
+        public static AspNetTicketSerializer Default { get; } = new AspNetTicketSerializer();
```

Comment:
Exposing a property using the vNext serializer was wrong, I took the liberty to fix it :smile: